locale/fr_CA/editor.xml

```diff
@@ -35,8 +35,6 @@
 	<message key="editor.notifyUsers.includeToc">Inclure le sommaire de ce numéro :</message>
 	<message key="editor.editorAdministration">Administration du rédacteur</message>
 	<message key="editor.issues.createIssue">Créer un numéro</message>
-	<message key="editor.issues.backIssues"> Numéros précédents</message>
```

Comment:
I removed editor.issues.backIssues because it has not is corresponding in locale/en_US/editor.xml.
This key 'editor.issues.backIssues' is actually in locale/en_US/locale.xml, therefore the french translation is in locale/fr_CA/locale.xml.